blueocean-web/src/main/js/try.js

```diff
@@ -1,12 +1,7 @@
 var $ = require('jquery-detached').getJQuery();
 
 function getAppUrl() {
-    const rootUrl = $('head').attr('data-rooturl');
-    if (!rootUrl) {
-        return '';
-    } else {
-        return rootUrl;
-    }
+    return rootUrl = $('head').attr('data-rooturl');
```

Comment:
I'm route the home directory path for Jenkins using nginx.

For example,
http://10.0.0.0/jenkins/ is home directory, data-rooturl value is, "/jenkins"

AS-IS,
when it contains the value, it is reset.
so, when I click the button, it will be linked "http://10.0.0.0/blue/"

TO-BE,
when it contains the value(ex, /jenkins/), it is maintain.
so, when I click the button, it will be linked "http://10.0.0.0/jenkins/blue/"